### PR TITLE
Animation graph support; 1D blending support; Modify animation test-case.

### PR DESCRIPTION
**Pull request by shrinktofit**

### CODE REVIEW

**Comment by shrinktofit** [comment]
This pull request solves issue #689.

File: index.js [hunk 1]
```diff
@@ -63,6 +63,7 @@ import input from './lib/input';
 
 import enums from './lib/platform/enums';
 import sys from './lib/platform/sys';
+import * as animation from './lib/framework/animation'
```
Comment (dabingnn) [inline index.js]: missed `;`

Comment (shrinktofit) [reply index.js]: Fixed.

**Comment by shrinktofit** [comment]
This pull request solves issue #685.

**Comment by dabingnn** [comment]
已经拆分pr

**Pull request closed without merging**